kasa/tests/test_usage.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+from kasa.modules import Usage
+
```

Comment:
If you don't mind, feel free to add tests for other methods in the usage module, too. The more, the merrier as it makes it harder to break the code / API contracts accidentally :-)

Reply:
I'd be happy to look at adding more tests but I think it requires extending newfakes etc to add the Usage module, which I think is beyond the scope of this PR. Also, I'm not quite sure I fully understand how the newfakes system is working yet, so I either need to put in a bit more time tracing what's going on or get some hints.